Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -8949,7 +8949,7 @@ void WebPage::frameWasFocusedInAnotherProcess(WebCore::FrameIdentifier frameID)
     auto* frame = WebProcess::singleton().webFrame(frameID);
     if (!frame)
         return;
-    m_page->focusController().updateFocusedFrame(frame->coreFrame());
+    m_page->focusController().setFocusedFrame(frame->coreFrame(), false);
```

Comment:
We generally try to avoid passing bools like this because it's hard to know what this means here. Prefer named enums.